site/en/_posts/2021-01-19-phase3.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+lang: en
+lang-ref: DFG Bewilligung
+title: "Phase III of the OCR-D-coordination project granted"
+tags: [DFG, grant, phase 3]
+author: Elisabeth Engl
+---
+
+The coordination project's application for the third phase of the OCR-D funding initiative was approved by the DFG.
+In phase III, the results of the previous MP phase will be stabilised and the productive use of the
```

Comment:
"Active" sounds better in English IMHO, i.e. "In phase III we will stabilise the results and initiate the productive use